### ISSUE TITLE
The crawler only crawled down 2k products, not all of Amazon's products.

### ISSUE THREAD

**Issue by wfh1300** [platonai/exotic-amazon#10]
![de757c3e453c8aa75e360c311909869](https://user-images.githubusercontent.com/39584730/221144545-066f8543-b5e8-4a56-828c-4b15f8f4c3e3.png)
Downloads are no longer changing at 2k. But Amazon's products are in the hundreds of millions.

**Comment by platonai** [comment]
Of course, our program will not crawl the entire amazon.com by default, which would be a very huge project. To start a real world data project, you need to make some simple configuration changes. Usually, each user has some special business logic, which requires a little code modifications to meet the business needs.

**Comment by wfh1300** [comment]
Would you please tell me which file should I change？

**Comment by platonai** [comment]
Update your code, repackage and run:

`ENV=prod java -jar target/exotic-amazon-0.0.2-SNAPSHOT.jar`

**Comment by platonai** [comment]
Essentially, a java environment variable named prod must be passed in to run the program in production mode.
